examples/ERP/classify_P300_bi.py

```diff
@@ -0,0 +1,144 @@
+
+warnings.filterwarnings("ignore")
+
+set_log_level("info")
+
+##############################################################################
+# Create Pipelines
+# ----------------
+#
+# Pipelines must be a dict of sklearn pipeline transformer.
+
+##############################################################################
+# We have to do this because the classes are called 'Target' and 'NonTarget'
+# but the evaluation function uses a LabelEncoder, transforming them
+# to 0 and 1
+labels_dict = {"Target": 1, "NonTarget": 0}
+
+paradigm = P300(resample=128)
+
+datasets = [bi2012()]  # MOABB provides several other P300 datasets
+
+# reduce the number of subjects, the Quantum pipeline takes a lot of time
+# if executed on the entire dataset
+n_subjects = 5
+for dataset in datasets:
+    dataset.subject_list = dataset.subject_list[0:n_subjects]
+
+overwrite = True  # set to True if we want to overwrite cached results
+
+pipelines = {}
+
+# A Riemannian Quantum pipeline provided by pyRiemann-qiskit
+# You can choose between classical SVM and Quantum SVM.
+pipelines["RG+QuantumSVM"] = QuantumClassifierWithDefaultRiemannianPipeline(
+    shots=None,  # 'None' forces classic SVM
+    nfilter=2,  # default 2
+    # default n_components=10, a higher value renders better performance with
+    # the SVM version used in quiskit
+    dim_red=PCA(n_components=10),
+    # q_account_token='' #IBM Quantum TOKEN
+    )
+
+# Here we provide a pipeline for comparison:
+
+# This is a standard pipeline similar to
+# QuantumClassifierWithDefaultRiemannianPipeline, but with LDA classifier
+# instead.
+pipelines["RG+LDA"] = make_pipeline(
+    # applies XDawn and calculates the covariance matrix, output it matrices
+    XdawnCovariances(
+        nfilter=2,
+        classes=[labels_dict["Target"]],
+        estimator="lwf",
+        xdawn_estimator="scm"
+    ),
+    TangentSpace(),
+    PCA(n_components=10),
+    LDA(solver="lsqr", shrinkage="auto"),  # you can use other classifiers
+)
+
+print("Total pipelines to evaluate: ", len(pipelines))
+
+evaluation = WithinSessionEvaluation(
+    paradigm=paradigm,
+    datasets=datasets,
+    suffix="examples",
+    overwrite=overwrite
+)
+
+results = evaluation.process(pipelines)
+
+print("Aaveraging the session performance:")
+print(results.groupby('pipeline').mean('score')[['score', 'time']])
+
+##############################################################################
+# Plot Results
+# ----------------
+#
+# Here we plot the results to compare the two pipelines
+
+fig, ax = plt.subplots(facecolor="white", figsize=[8, 4])
+
+sns.stripplot(
+    data=results,
+    y="score",
+    x="pipeline",
+    ax=ax,
+    jitter=True,
+    alpha=0.5,
+    zorder=1,
+    palette="Set1",
+)
+sns.pointplot(data=results,
+              y="score",
+              x="pipeline",
+              ax=ax, zorder=1,
+              palette="Set1")
+
+ax.set_ylabel("ROC AUC")
+ax.set_ylim(0.3, 1)
+
+fig.show()
```

Comment:
```suggestion
plt.show()
```

The figure is not shown on the generated documentation, try using `plt.show()` to display the plot.